### ISSUE TITLE
ICLR '20 | Large Batch Optimization for Deep Learning: Training BERT in 76 minutes 

### ISSUE THREAD

**Issue by zhongyuchen** [zhongyuchen/read-papers#3]
https://openreview.net/attachment?id=Syx4wnEtvH&name=original_pdf

**Comment by zhongyuchen** [comment]
https://openreview.net/forum?id=Syx4wnEtvH
review

**Comment by zhongyuchen** [comment]
本质上就是Adam加了个normalize……然后说有双重normalize 
1. 自带的处以 sqrt(v) ... per parameter normalize. 
2. 加的layer-wise normalize

然后通过一系列实验证明LAMB比LARS更好. 还给了理论证明

![image](https://user-images.githubusercontent.com/25879526/72266582-a0717200-3659-11ea-838a-197276aadc8f.png)

之前SOTA算法是LARS，成功运用在了imagenet上，但是他说bert不行，scale到32k就爆炸了. 

这篇文章问题其实很多
1. 只看了SQuAD数据集，bert可远不止测了SQuAD...所以你到底有没有训练出来一个bert还不知道. pretraining和finetuning之间是有gap的. 
2. 给出的理论表明似乎lamb反而要比lars收敛的慢(see https://openreview.net/forum?id=Syx4wnEtvH&noteId=H1eSuSc9YS) 


我的疑惑，为什么 lr突然正比于 sqrt(batch_size)了，之前不都是linear吗？？？

**Comment by zhongyuchen** [comment]
另外里面说\beta_1 = \beta_2 = 0的时候，退化成signSGD #13  ，我很喜欢.

**Comment by zhongyuchen** [comment]
突然发现BERT也是用adam。。。。那这个思路很正常了。。。。
